src/iv/ivgl.cpp

```diff
@@ -917,70 +999,427 @@ IvGL::paint_pixelview()
+        std::string normalized_header;
+        std::string empty_normalized_header;
+        if (maxLengths.normalized > 0) {
+            normalized_header = Strutil::fmt::format("{:<{}}  ", "Norm",
+                                                     maxLengths.normalized);
+            empty_normalized_header
+                = Strutil::fmt::format("{:<{}}  ", "    ",
+                                       maxLengths.normalized);
+        } else {
+            normalized_header       = "";
+            empty_normalized_header = "";
+        }
+
+        // Print Norm, Min, Max column headers with normal white color
+        std::string base_header = Strutil::fmt::format(
+            "   {:<{}}  {}{:<{}}  {:<{}}  {:<{}}  ", "   ",
+            maxLengths.centerValue, normalized_header, "Min", maxLengths.min,
+            "Max", maxLengths.max, "   ", maxLengths.avg);
+        shadowed_text(x_text, y_text, 0.0f, base_header, normal_text_color);
+
+        // Print "Val" column headers with normal cyan color
+        std::string val_header = Strutil::fmt::format(
+            "   {:<{}}  {}{:<{}}  {:<{}}  {:<{}}  ", "Val",
+            maxLengths.centerValue, empty_normalized_header, "   ",
+            maxLengths.min, "   ", maxLengths.max, "   ", maxLengths.avg);
+        shadowed_text(x_text, y_text, 0.0f, val_header,
+                      center_pix_value_text_color);
+
+        // Print "Avg" column header with normal yellow color
+        std::string avg_header = Strutil::fmt::format(
+            "   {:<{}}  {}{:<{}}  {:<{}}  {:<{}}  ", "   ",
+            maxLengths.centerValue, empty_normalized_header, "   ",
+            maxLengths.min, "   ", maxLengths.max, "Avg", maxLengths.avg);
+        shadowed_text(x_text, y_text, 0.0f, avg_header, avg_value_text_color);
+
+        y_text += text_line_height;
+    }
+
+    for (const auto& stat : channels_stats) {
+        // Build the "Norm" column header conditionally
+        std::string normalized_col;
+        if (maxLengths.normalized > 0) {
+            normalized_col = Strutil::fmt::format("{:<{}}  ", stat.normalized,
+                                                  maxLengths.normalized);
+        } else {
+            normalized_col = "";
+        }
+
+        std::string line
+            = Strutil::fmt::format("{:<{}}: {:<{}}  {}{:<{}}  {:<{}}  {:<{}}  ",
+                                   stat.name, maxLengths.name, stat.centerValue,
+                                   maxLengths.centerValue, normalized_col,
+                                   stat.min, maxLengths.min, stat.max,
+                                   maxLengths.max, stat.avg, maxLengths.avg);
+
+        QColor channelColor;
+        if (stat.name[0] == 'R') {
+            channelColor = QColor(255, 50, 50);
+        } else if (stat.name[0] == 'G') {
+            channelColor = QColor(100, 255, 90);
+        } else if (stat.name[0] == 'B') {
+            channelColor = QColor(107, 188, 255);
```

Comment:
To my old eyes, the blue (which is definitely blue-ish, but not super saturated) is very readable and pleasant. The green somewhat less so, and the red somewhat straining to read because it's so saturated and with high contrast. Maybe make the G and B colors also a bit less saturated?

Reply:
Amendment: your colors actually look pretty good when the image is black or quite dark, but when the image is fairly light, saturated-red-on-white is harder to read.

Reply:
And we haven't even got into territory of a11y :-D Alright. let me play with that. and see what I came out with that would be better. I did spent a bit of time trying to find balance between saturation and lightness, but more can be done, I agree.

Reply:
![Screenshot 2025-07-13 at 6 04 09 pm](https://github.com/user-attachments/assets/11b491eb-afa9-4ce3-baa9-d4e9db6ce4b5)
what I came up with so far. They are farily desaturated (like blue is) but prettu light.